hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/config/EntityNumbers.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.hedera.node.app.service.file.impl.config;
+
+import com.hedera.hapi.node.base.AccountID;
+import com.hedera.hapi.node.base.ContractID;
+import com.hedera.hapi.node.base.FileID;
+import com.hedera.node.app.spi.numbers.HederaAccountNumbers;
+import com.hedera.node.app.spi.numbers.HederaFileNumbers;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+@Singleton
+public class EntityNumbers {
+    public static final long UNKNOWN_NUMBER = Long.MIN_VALUE;
+
+    private final HederaFileNumbers fileNumbers;
+    private final HederaNumbers hederaNumbers;
+    private final HederaAccountNumbers accountNumbers;
+
+    @Inject
+    public EntityNumbers(
+            final HederaFileNumbers fileNumbers,
+            final HederaNumbers hederaNumbers,
+            final HederaAccountNumbers accountNumbers) {
+        this.fileNumbers = fileNumbers;
+        this.hederaNumbers = hederaNumbers;
+        this.accountNumbers = accountNumbers;
+    }
+
+    public boolean isSystemFile(FileID id) {
```

Comment:
Add @NonNull and final to method parameters in this class

Reply:
done thanks